VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-waiting-room/judge-waiting-room.component.ts

```diff
@@ -22,13 +22,16 @@ import { NotificationToastrService } from '../services/notification-toastr.servi
 import { RoomClosingToastrService } from '../services/room-closing-toast.service';
 import { VideoCallService } from '../services/video-call.service';
 import { WaitingRoomBaseDirective } from '../waiting-room-shared/waiting-room-base.component';
+import { DatePatternOption } from 'src/app/services/models/date-patterns';
 
 @Component({
     selector: 'app-judge-waiting-room',
     templateUrl: './judge-waiting-room.component.html',
     styleUrls: ['./judge-waiting-room.component.scss', '../waiting-room-global-styles.scss']
 })
 export class JudgeWaitingRoomComponent extends WaitingRoomBaseDirective implements OnInit, OnDestroy {
+    public datePatternOption = DatePatternOption;
```

Comment:
I am fairly new in angular, but shouldn't this be an interface and then the value could be set in the class? 

Or was it already considered and you would have repetition so instead enum was used?

Reply:
The choice of enum is the right one as a DatePatternOption variable is one of x  date pattern options.

However, this line is not required. As I understand, this line is here so you can do 

`this.datePatternPipe = this.datePatternOption.Date;` 

on line 86, but you can just do

`this.datePatternPipe = DatePatternOption.Date;` 

If you could change that and the other files which use this pattern that would get rid of some unneeded lines. If I've missed something please let me know 👍

Reply:
yea you can argue that we can set the value of props in an enum to be however we like, I am just used to seeing it with generic number values.

Reply:
I will remove the enum options as we just need to pass the string pattern on the Pipe...this was over complicating things itself